segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeperstorage/BookKeeperStorageMetrics.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage.impl.bookkeeperstorage;
+
+import io.pravega.shared.MetricsNames;
+import io.pravega.shared.metrics.Counter;
+import io.pravega.shared.metrics.MetricsProvider;
+import io.pravega.shared.metrics.OpStatsLogger;
+import io.pravega.shared.metrics.StatsLogger;
+
+final class BookKeeperStorageMetrics {
```

Comment:
Awesome, thanks for adding it.

Reply:
Thanks. Added code now to actually update the metrices :D.